src/garage/np/algos/rl_algorithm.py

```diff
@@ -9,10 +9,17 @@ class RLAlgorithm(abc.ABC):
         If the field sampler_cls exists, it will be by LocalRunner.setup to
         initialize a sampler.
 
+    Args:
+        env_spec (garage.envs.EnvSpec): Environment specification.
+        discount (float): Discount.
```

Comment:
By adding two more fields, you've basically doubled the size of one of the most commonly used APIs in garage. Keeping this API small is very important, so please remove these fields.